learn/cloud/analytics.mdx

```diff
@@ -0,0 +1,197 @@
+
+### Searches with no results and no result rate
+
+Searches with no results indicates search terms that did not return any results. If multiple queries correspond to the same [search id](#total-searches-and-search-ids), Meilisearch Cloud only tracks the last query associated with it.
+
+The no result rate is the number of searches with no results divided by the total number of searches.
+
+### Conversion rate
+
+A conversion is when users take action relevant to your business beyond clicking on a search result. Exactly what counts as a conversion depends on your specific application. For example, a web shop might count adding a product to their shopping cart as a conversion.
+
+Conversion rate indicates the number of searches that resulted in a `conversion` event divided by the total number of searches.
+
+To track conversion rates, [configure your application to send `conversion` events to Meilisearch Cloud](#configuring-conversion-rate).
+
+### Click-through rate and average click position
+
+Click-through rate is the number of times users have clicked on a search result divided by the total number of searches.
+
+Average click position indicates the average position of the search results users clicked on, with 0 indicating the first search result. 
+
+In both metrics, lower values indicate searches are returning relevant results.
+
+To track click-through rate and average click position, [configure your application to send `click` events to Meilisearch Cloud](#configuring-click-through-rate-and-average-click-position). 
+
+### Search requests
+
+Indicates the total amount of queries targeting the `/search` route. This count is not dependent on search IDs and includes all search queries.
+
+### Search latency
+
+The amount of time between a user submitting a search request and receiving search results in return. Lower numbers indicate the interface is responding quickly, leading to a better search experience.
+
+### Countries with most searches
+
+The number of searches per country, with the countries with biggest amount of searches at the top of the list.
+
+## Deactivating analytics
+
+### Disable analytics in the project overview
+
+Log into your Meilisearch Cloud account and navigate to your project's overview. Find the "Analytics" section and press the "Disable analytics" button:
+
+![The analytics section of the project overview. It shows one button, "Disable analytics", and a short explanation of the feature.](https://raw.githubusercontent.com/meilisearch/documentation/cloud-analytics-guide/assets/images/cloud-analytics/02-analytics-disable.png)
+
+### Update URLs in your application
+
+Disabling analytics changes your project URL to its previous value. Update your application so all API requests point to the correct URL:
```

Comment:
FYI, the same thing happens when they enable analytics, the edge.meilisearch will still work even after disabling, the disabling only affects the front-end, so the user has no disruption in their service. I don't know if you want to mention that, but it is just to let you know.

The only thing that will never work, is the user trying the edge without enabling the analytics at least once!